soyhtml/exec.go

```diff
@@ -330,6 +330,33 @@ func (s *state) evalPrint(node *ast.PrintNode) {
 		}
 	}
 
+	for _, directiveName := range ObligatoryPrintDirectiveNames {
```

Comment:
Could you instead append this to the list of node directives on #302 to avoid duplicating the code?

Reply:
I can do that. It feels a little weird because I have to turn it into a Node which has a Position but the directive doesn't actually appear in the input anywhere. I'll just give it the same position as the PrintNode that it belongs to, I don't think it's too important.